api/v1/ytsaurus_types.go

```diff
@@ -571,6 +571,13 @@ type CommonSpec struct {
 	//+kubebuilder:default:=false
 	//+optional
 	UseIPv4 bool `json:"useIpv4"`
+	//+kubebuilder:default:=true
+	//+optional
+	KeepSocket bool `json:"keepSocket"`
+	//+kubebuilder:default:=false
+	//+optional
+	ForceTcp bool `json:"forceTcp"`
```

Comment:
I think better call it `ForceTCP` (and in tag also) according to go [style guides ](https://go.dev/wiki/CodeReviewComments#initialisms) similarly to HTTP even if linter doesn't triggered on that.

Reply:
> ForceTCP

```bash
$ make lint
api/v1/ytsaurus_types.go:579:17: json(camel): got 'forceTCP' want 'forceTcp' (tagliatelle)
        ForceTCP *bool `json:"forceTCP,omitempty"`
                       ^
make: *** [lint] Error 1
```


add like - 
```
	//+kubebuilder:default:=false
	//+optional
	//nolint:tagliatelle
	ForceTCP *bool `json:"forceTCP,omitempty"`
```
?

Reply:
Let's just stick with what linter wants in json tag (`forceTcp `) then.

We might configure tagliatelle  linter with `goCamel` instead of `camel`, but I suppose it will want other fields renamed which is not backward compatible.